Kickstarter-iOS/Features/ChangeEmail/Controller/ChangeEmailView.swift

```diff
@@ -0,0 +1,195 @@
+import Library
+import SwiftUI
+
+enum FocusField {
+  case newEmail
+  case currentPassword
+}
+
+@available(iOS 15.0, *)
+struct ChangeEmailView: View {
+  @State var emailText: String
+  @State private var newEmailText = ""
+  @State private var passwordText = ""
```

Comment:
I know you don't have the view model piece wired up yet, but one alternative to having these be @State properties in the View would be to instead define them as @Published properties directly in the view model. That way we could keep it all in the VM and ready to be used there, rather than having to pass it back in a method from the View.